VSB cleanup for OADP 1.1 (#59894)
openshift/openshift-docs: modules/oadp-deleting-cluster-resources-following-success.adoc

// Module included in the following assemblies:
//
// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

:_content-type: PROCEDURE
[id="oadp-deleting-cluster-resources-following-success_{context}"]
= Deleting cluster resources following a successful backup and restore using Data Mover

> You can delete any `volumeSnapshotBackup/volumeSnapshotRestore` CRs that remain in your application namespace after a successful backup and restore in which you used Data Mover.

Reviewer: * Should avoid using a slash
* Usually CRs start with a capital, should these be capital V's?

```suggestion
You can delete any `VolumeSnapshotBackup` or `VolumeSnapshotRestore` CRs that remain in your application namespace after a successful backup and restore in which you used Data Mover.
```

Author: OK, that was just ... dumb

.Procedure

> . Delete cluster resources that remain after a backup in which you use Data Mover:

Reviewer: "in which" sounds really stiff/proper. Could you use "where you used Data Mover" throughout this file?

```suggestion
. Delete cluster resources that remain after a backup where you used Data Mover:
```

+
[source,terminal]
----

> $ oc delete vsb -n <app-ns> --all

Reviewer: underscore for replaceable values: <app_ns>

Or really I'd recommend writing out ns to make it more obvious that ns is namespace: <app_namespace>

Same for below

----

. Delete cluster resources that remain after a restore in which you use Data Mover:
+
[source,terminal]
----
$ oc delete vsr -n <app-ns> --all
----

> . If needed, delete any `volumeSnapshotContent` resources that remain after a backup and restore in which you use Data Mover:

Reviewer: Same question on if `VolumeSnapshotContent` should be capitalized
